whois-commons/src/main/resources/patch/default-maintainer-history.sql

```diff
@@ -0,0 +1,2 @@
+DROP TABLE IF EXISTS `default_maintainer_history`;
```

Comment:
I think we need to version patch files against the current or next version of Whois, also update the `version` table to match - check the other patch files for examples